.gitignore

```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
```

Comment:
I'd rather see unrelated things like this in a separate PR so they can be discussed properly, but I'm not going to make you change it now.

Reply:
I understand. I included this other change after git initially grabbed a bunch of unwanted, IDE-related files on my first attempt. I could have redone a more selective "git add", but felt it was cleaner to do handle it right-away in .gitignore. Felt a bit overkill to do a separate pull request just for it.

Reply:
> Felt a bit overkill to do a separate pull request just for it.

I understand. While it can seem like overkill when you're learning how to do pull requests, once you get used to them, they should feel less "heavy". Each PR should be a self-contained fix or improvement, even if it is tiny.

Reply:
Got it.